### DISCUSSION TITLE
Flashing STM32F4 by drag-and-drop binary file MSC

### DISCUSSION THREAD

**Discussion by jadczakoff** [hathach/tinyusb#1699] [Q&A]
Hello,

I'm trying to make a flasher via mass storage class. The first problem is I cannot increase the amount of disk. I was changing this variable "DISK_BLOCK_NUM  = 16, // 8KB is the smallest size that windows allow to mount" but disk size is not changed. I'm trying to change it because my binary file is larger than the disk size. The general question is How can I do this flasher via MSC in STM32F4 ? Is it possible ?

Link to project: https://github.com/jadczakoff/Deaf_band_v4

**Comment by hathach** [comment]
check out this project, this should have everything (most) working as bootloader https://github.com/adafruit/tinyuf2
